### PR TITLE
fix prometheus integration

### PR DESCRIPTION
**Pull request by paulocoutinhox**
If it is ok, can you generate a new version to us? Thanks.

### CODE REVIEW

File: django_q/cluster.py [hunk 1]
```diff
@@ -231,7 +232,10 @@ def reincarnate(self, process):
             )
         else:
             if prometheus_multiprocess:
-                prometheus_multiprocess.mark_process_dead(process.pid)
+                prometheus_path = os.getenv("PROMETHEUS_MULTIPROC_DIR")
```
Comment (GDay) [inline django_q/cluster.py:235]: Implementation looks good. Thanks!

I am a nitpicking here, but could you make this a one liner (so both conditions in one if statement) and then add a comment to indicate that we check this to make sure the library is configured?

Comment (paulocoutinhox) [reply django_q/cluster.py]: Done

Comment (GDay) [reply django_q/cluster.py]: Thanks! I just noticed that there is also a lower case environment variable (same name) that gets accepted to, so I have added that one too. Didn't want to bother you again!
I will get this merged.

**Review by GDay — APPROVED** [review]
LGTM! Thanks again!

**Pull request merged**